website/docs/docs/deploy/deployment-overview.md

```diff
@@ -44,9 +44,9 @@ Learn how to use dbt Cloud's features to help your team ship timely and quality
     icon="dbt-bit"/>
 
 <Card
-    title="Merge jobs"
-    body="Set up jobs to run when pull requests are merged in your Git repository."
-    link="/docs/deploy/merge-jobs"
```

Comment:
Do we need a redirect for this?

Reply:
good eyes! 

nah, not in this case. the "merge jobs" page still exists, it didn't get deleted.